src/containers/Home/index.js

```diff
@@ -2,6 +2,8 @@ import React, { Component } from 'react';
 import { message } from 'antd';
 import propTypes from 'prop-types';
 import HomePage from './home';
+import { withFirebase } from '../Firebase/index';
+
 import { journalsData, username, usergoal } from './staticData';
```

Comment:
```suggestion
import { journalsData, username, userGoal } from './staticData';
```

Reply:
This is not my work , I have just added firebase part.